rust-src/concordium_base/src/updates.rs

```diff
@@ -794,8 +794,6 @@ pub struct CreatePlt {
     pub token_id:                  protocol_level_tokens::TokenId,
     /// A SHA256 hash that identifies the token module implementation.
     pub token_module:              protocol_level_tokens::TokenModuleRef,
-    /// The address of the account that will govern the token.
-    pub governance_account:        AccountAddress,
```

Comment:
Can you please add to your description in your PR links to related PRs that will help reviewers to easily get an overview/jump to associated PRs.